hs/src/Reach/Connector/ETH_Solidity.hs

```diff
@@ -243,7 +243,7 @@ data SolCtxt = SolCtxt
   , ctxt_plo :: PLOpts
   , ctxt_intidx :: Counter
   , ctxt_ints :: IORef (M.Map Int Doc)
-  , ctxt_outputs :: IORef (M.Map DLVar Doc)
+  , ctxt_outputs :: IORef (M.Map String Doc)
```

Comment:
Use the outputEvent name which is either `oe${DLVar}` or the Event label. This prevents multiple event declarations from being created when emitting an Event multiple times in a Reach program.